### ISSUE TITLE
Apple Magic Keyboard: [^] and [<] keys are swapped

### ISSUE THREAD

**Issue by DennyWeinberg** [christian-korneck/win10-keyboard-macbook-german#2]
Hi,

I've read "Apple Magic Keyboard: [^] and [<] keys are swapped" in your readme but I  don't understand the solution.
Below the escape I would like to have §° and next to the shift <>. But they are swapped, already in Mac OS. That issue has nothing to do with your (awesome) keyboard layout for Windows, but do you know how to fix that?
I'm using the Swiss German keyboard layout in Mac OS.
Here are my keys:
```
SHIFT: >+"*ç%&/()=?`
OPTION: ≤±“#Ç[]|{}≠¿´
```

All are correct except the first one.

Thanks!

Denny

**Comment by christian-korneck** [comment]
how do you access Windows on your Mac? Bare metal (bootcamp), via Remote Desktop (RDP), VNC, etc?

**Comment by DennyWeinberg** [comment]
MS RDP client. But as I sais, its already false on my mac. That's strange. As mac would not have the right layout for their own Macbook. That somehow doesn't make sense. It's not a big issue because only two keys are swapped. Buy still annoying.

**Comment by christian-korneck** [comment]
Would it work for you to just swap the keys in your .klc file? This seems to have worked for another user. (The custom .klc file (with compiled installer) that I've created for him is here:

[DeApplC1.klc.txt](https://github.com/christian-korneck/win10-keyboard-macbook-german/files/6340658/DeApplC1.klc.txt)
[deapplc1.zip](https://github.com/christian-korneck/win10-keyboard-macbook-german/files/6340659/deapplc1.zip)

**Comment by christian-korneck** [comment]
also I believe you can remap keys in MacOS:
https://developer.apple.com/library/archive/technotes/tn2450/_index.html

**Comment by DennyWeinberg** [comment]
Yes I could with Karabiner. But I don't get why it doesn't match with a default keyboard layout. Anyway, thanks for your help!

**Comment by christian-korneck** [comment]
> But I don't get why it doesn't match with a default keyboard layout. 

I don't know. :-( 
If you find out, please share.
